tests/ext/drv/test_ext.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/*++
+
+Abstract:
+
+   Header file for structures/prototypes of the driver.
+
+
+Environment:
+
+    Kernel mode
+
+--*/
+
+#pragma once
+
+#include <ntddk.h>
+#include "ebpf_platform.h"
+
+typedef struct _test_program_context test_program_context_t;
+
+/**
+ * @brief Register program information providers with eBPF core.
```

Comment:
```suggestion
 * @brief Register program information providers.
```
Also above says "providers" plural but the function has "provider" singular.  Which is correct?  Update the incorrect one.